script/deployment/DeploymentScript.s.sol

```diff
@@ -497,12 +496,25 @@ contract DeploymentScript is Script {
         console2.log(
             "-----------------------------------------------------------------------------"
         );
-        console2.log("Deploy factory implementations \n");
+        console2.log("Setup last implementations \n");
```

Comment:
Not sure exactly what this comment means, as a proxy is deployed directly below it?

Reply:
Yeah the comment was kinda bad
Changed it